src/mmw/apps/user/views.py

```diff
@@ -381,3 +382,21 @@ def hydroshare_auth(request):
                                'possibly misconfigured', 'warning')
 
     return render_to_response('user/hydroshare-auth.html', context)
+
+
+@decorators.api_view(['POST'])
+@decorators.permission_classes((IsAuthenticated, ))
+def hydroshare_logout(request):
```

Comment:
We are deleting from two different tables so we should decorate this with `@transaction.atomic` so that the deletes succeed or fail together.

Reply:
Ah great idea, I didn't know about that!